v3/views/exercise/readme.pg

```diff
@@ -1,3 +1,8 @@
 node exercise: problem do
-  attributes :track_id, :language, :slug, :name, :readme, :blurb
+  attributes :track_id, :language, :slug
+  if problem.exists?
```

Comment:
If it doesn't exist then it's a null problem, and it should have some reasonable zero values (empty strings at the very least). I don't want to have a conditional in this view.